templates/shit-o-lator.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Shit-o-lator v.4.1!!! ;)</title>
+    <link rel="stylesheet" type="text/css" href="/static/css/style.css">
```

Comment:
From flask tutorial:
To generate URLs for static files, use the special 'static' endpoint name:
url_for('static', filename='style.css')